apps/wallet/src/ui/app/hooks/accounts-v2/useAccounts.ts

```diff
@@ -14,7 +14,7 @@ export function useAccounts() {
 		queryFn: () => backgroundClient.getStoredEntities<SerializedUIAccount>('accounts'),
 		cacheTime: 30 * 1000,
 		staleTime: 15 * 1000,
-		refetchInterval: 5 * 1000,
+		refetchInterval: 30 * 1000,
```

Comment:
this is being invalidated on events from service worker (events related to accounts) - just to make sure we update them in case we miss any event but probably not necessary. We can revisit this